docs/reference/indices/clone-index.asciidoc

```diff
@@ -49,20 +39,44 @@ PUT /my_source_index/_settings
   }
 }
 --------------------------------------------------
-// TEST[continued]
+// TEST[s/^/PUT my_source_index\n/]
 
 <1> Prevents write operations to this index while still allowing metadata
     changes like deleting the index.
 
-[float]
-==== Cloning an index
+
+[[clone-index-api-desc]]
+==== {api-description-title}
+
+Use the clone index API
+to clone an existing index into a new index,
+where each original primary shard is cloned
+into a new primary shard in the new index.
+
+[[cloning-works]]
+===== How cloning works
+
+Cloning works as follows:
+
```

Comment:
When I see "as follows", I tend to want to rewrite the intro so it isn't needed. :-) Not necessary for this PR, but here I'd tweak the intro and modify the steps so that "it" has a clear antecedent. And since this is a list, I'd get rid of the "first", "next", "finally" in the steps. (A numbered list is probably in order.)

Reply:
Thanks. I'll make a note. This just relocates existing content from https://github.com/elastic/elasticsearch/pull/46762/files#diff-9ce2f43035644d29a3e773dda62b27e0L9